sass/components/_file.scss

```diff
@@ -86,10 +86,8 @@ Style guide: file.1
 }
 
 .file__btn {
-  @extend .col-md-4;
-  @extend .col-sm-4;
-  @extend .col-xs-12;
-
+  width: 33.3%;
+  text-align: center;
```

Comment:
Well, i love the fact that we move out of a bootstrap grid in order to define the width of this element, @gingapa, but this way we change the responsive behavior, meaning that in mobile we used to have a 100% button and now we have it for all the viewport dimensions  as a 33,3% wide button. Would you reproduce also this responsive behavior without bootstrap dependencies?